server/config.js

```diff
@@ -1,9 +1,12 @@
 import { join } from 'path'
-import { readFile } from 'mz/fs'
+import { existsSync } from 'fs'
 
 const cache = new Map()
 
-const defaultConfig = { cdn: true }
+const defaultConfig = {
+  cdn: true,
```

Comment:
With #414 now we don't have an option called `cdn`. So we don't need this anymore.